Project/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -1,16 +1,36 @@
+using MLAgentsExamples;
 using UnityEngine;
 using Unity.MLAgents;
 using Unity.MLAgentsExamples;
 using Unity.MLAgents.Sensors;
+using BodyPart = Unity.MLAgentsExamples.BodyPart;
 
 public class WalkerAgent : Agent
 {
+    [Header("Walking Speed")]
```

Comment:
The only public variable after this header is `maximumWalkingSpeed`. I think this header will be confusing.

Reply:
removed the header. when we have more settings to control the walking behavior we can revisit using a header here.